packages/trader/src/Stores/client-store.js

```diff
@@ -868,6 +868,25 @@ export default class ClientStore extends BaseStore {
         });
     }
 
+    @action.bound
+    onResetPassword(password, onResetComplete, formik_actions) {
```

Comment:
should this be in the client-store?

Reply:
i ollowed the account signup way, where there is onSignup in client store, but since its specific only for reset-password use, i think we dont need to have action in client store for this. especially we dont use any other action inside client store